smtp-crawler/src/main/java/be/dnsbelgium/mercator/smtp/domain/crawler/SmtpAnalyzer.java

```diff
@@ -110,12 +106,12 @@ private SmtpVisitEntity doCrawl(String domainName) {
         for (MXRecord mxRecord : mxLookupResult.getMxRecords()) {
           logger.debug("mxRecord = {}", mxRecord);
           String hostName = mxRecord.getTarget().toString(true);
-          Optional<List<SmtpHostEntity>> hosts = createSmtpHosts(hostName, mxRecord.getPriority());
+          Optional<List<SmtpHost>> hosts = createSmtpHosts(hostName, mxRecord.getPriority());
           if (hosts.isPresent()) {
-            for(SmtpHostEntity host : hosts.get()){
+            for(SmtpHost host : hosts.get()){
               host.setFromMx(true);
               // TODO: check error
```

Comment:
Should we read this error? What do we want to do with it if there's an error here?